staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5165,6 +5175,12 @@ type SecurityContext struct {
 	// PodSecurityContext, the value specified in SecurityContext takes precedence.
 	// +optional
 	RunAsUser *int64 `json:"runAsUser,omitempty" protobuf:"varint,4,opt,name=runAsUser"`
+	// The GID to run the entrypoint of the container process.
+	// Defaults to group specified in image metadata if unspecified.
```

Comment:
@pmorie is "Defaults to group specified in image metadata if unspecified." accurate? or does it default to gid 0?

Reply:
I see this is described more fully in another field: "Defaults to group specified in image metadata if unspecified. If group is not specified in image metadata, the default is GID 0."

is that runtime-specific, or an API guarantee?

Reply:
Pretty sure that's runtime specific. I would just say something along the lines of "Uses runtime default if unset".